### PR TITLE
Assets Go

### PR DESCRIPTION
**Pull request by unanoc**

### CODE REVIEW

**Review by scotts2080 — COMMENTED** [review]
- @Scotts2080****

File: go.sum [hunk 1]
```diff
@@ -278,6 +278,16 @@ github.com/trustwallet/assets-go-libs v0.0.6-0.20211124104413-520938a5b242 h1:7i
 github.com/trustwallet/assets-go-libs v0.0.6-0.20211124104413-520938a5b242/go.mod h1:0nQmM5unJu0GuWSFJrWgSyOi2uUGnYmjQ53SkKe7/qI=
 github.com/trustwallet/assets-go-libs v0.0.6 h1:a0hJXemtyya4do32CTGHHJZDVHSR9ql+P7HpLilo0A0=
 github.com/trustwallet/assets-go-libs v0.0.6/go.mod h1:0nQmM5unJu0GuWSFJrWgSyOi2uUGnYmjQ53SkKe7/qI=
+github.com/trustwallet/assets-go-libs v0.0.7-0.20211124164500-003dab6b2916 h1:TXr+Tk9Y4SxPEVn/CYNu4nFpOsvDecRFEfiZJDzVnDU=
+github.com/trustwallet/assets-go-libs v0.0.7-0.20211124164500-003dab6b2916/go.mod h1:0nQmM5unJu0GuWSFJrWgSyOi2uUGnYmjQ53SkKe7/qI=
+github.com/trustwallet/assets-go-libs v0.0.7-0.20211124170118-ab37fc3e5b48 h1:0B7BitT7BHBfa+vXKM8y9NLJ/RZ5cBhAvB052esXItQ=
+github.com/trustwallet/assets-go-libs v0.0.7-0.20211124170118-ab37fc3e5b48/go.mod h1:0nQmM5unJu0GuWSFJrWgSyOi2uUGnYmjQ53SkKe7/qI=
+github.com/trustwallet/assets-go-libs v0.0.7-0.20211124170607-cfb99cf6f96c h1:abu6Ek0/grRcyZt7PtnKTgP5hRZ707bF+uKy8c3r+2U=
```
Comment (Taylorman3) [inline go.sum:285]: Thanks

Comment (9764319764319764) [reply go.sum:285]: hamed

Comment (Taylorman3) [reply go.sum:285]: What Now

**Pull request merged**